modules/exploits/unix/http/zivif_ipcheck_exec.rb

```diff
@@ -0,0 +1,88 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
```

Comment:
Is there a reason for only `NormalRanking` ? Is there a risk of rebooting the device or preventing further exploitation?

It looks like a generic command execution in what I'm guessing is the ping interface. Probably:

```suggestion
  Rank = ExcellentRanking
```